cinema/models.py

```diff
@@ -1,10 +1,33 @@
 from django.db import models
 
 
+class Actor(models.Model):
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return f'{self.first_name} {self.last_name}'
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+
+    def __str__(self):
+        return f'{self.name}'
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
+
+
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
+    actors = models.ManyToManyField(Actor, related_name="actor")
+    genres = models.ManyToManyField(Genre, related_name="genres")
```

Comment:
wrong related name

Reply:
why you don't exclude setting.py in flake8, why i should reformate strings that were generated by django ?